Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -106,6 +106,9 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
                     input.put("merchantProfileId", venmoProfileId);
                     input.put("customerClient", "MOBILE_APP");
                     input.put("intent", "CONTINUE");
+
+                    input.putOpt("displayName", request.getDisplayName());
```

Comment:
In our iOS PR - we aren't adding the "displayName" if it isn't set. It probably doesn't matter / the graphQL api should be able to handle either, but figured I'd bring it up.

Reply:
Definitely yeah good callout. In this case if `getDisplayName()` returns null it should no-op.